src/app/GitExtUtils/GitUI/Interops/RECT.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.InteropServices;
+
+namespace System;
+
+internal static partial class NativeMethods
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct RECT
```

Comment:
Seems to be (kind of) a duplicate of `src\app\GitUI\Interops\RECT.cs`.

Reply:
This is a second time in my experience when interop needs to happen in `GitExtUtils` but most of it is located in `GitUI`. One way to solve this is to move everything to `GitExtUtils` but then it would need to be made public. Until then we are left with making duplicates.
To clarify, I can't use `src\app\GitUI\Interops\RECT.cs` in `GitExtUtils` project.

Reply:
> This is a second time in my experience when interop needs to happen in `GitExtUtils` but most of it is located in `GitUI`. One way to solve this is to move everything to `GitExtUtils` but then it would need to be made public. Until then we are left with making duplicates. To clarify, I can't use `src\app\GitUI\Interops\RECT.cs` in `GitExtUtils` project.

What do you think, @RussKie?